README.md

```diff
@@ -39,6 +39,10 @@ const server = new ProxyChain.Server({
     // Port where the server will listen. By default 8000.
     port: 8000,
 
+    // Hostname where the server will listen.
+    // optional, defaults to unspecified IP address, see e.g. https://en.wikipedia.org/wiki/IPv6_address#Unspecified_address
```

Comment:
Let's change it to "host", as it might be also an IP address. Plus can we improve the description:

> Optional host where the proxy server will listen. If not specified, there sever listens on an unspecified IP address (0.0.0.0 in IPv4, :: in IPv6). You can use this option to limit the access to the proxy server.

Reply:
yeah, true, good point